### ISSUE TITLE
[BUG] When using llama.cpp, traces missing retriever spans

### ISSUE THREAD

**Issue by mikeldking** [Arize-ai/phoenix#2218]
Question 2:  As a second issue, as noted in the id field above, our local LLM is somehow defaulting to ChatCompletion, instead of the Completion trace that we get when running, say, GPT3.5. This might be something we need to address through LlamaIndex, but there’s still an issue on the Phoenix side:
Whereas Completion creates a Query/Retrieval span where all retrieved documents can be recovered through Phoenix, ChatCompletion doesn’t allow access to any retrieved documents, as far as I can tell.
In other words, get_retrieved_documents(px.active_session()) returns an empty dataframe (with only column headers showing) when using ChatCompletion.
Is this expected behavior? Is there a different way to access the retrieved documents when using ChatCompletion?

**Comment by mikeldking** [comment]
Cannot reproduce. original thread here https://arize-ai.slack.com/archives/C04R3GXC8HK/p1707252784563709
